interactive/run.py

```diff
@@ -119,13 +129,13 @@ def write_files(checkout, analysis_request, codelist):
 
 
 def commit_and_push(checkout, analysis_request):
-    msg = f"Interactive analysis of codelist {analysis_request.codelist} ({analysis_request.id})"
+    msg = f"{analysis_request.id}: codelist {analysis_request.codelist} by {analysis_request.user.email}"
     email = analysis_request.user.email
     git("add", "project.yaml", "codelist.csv", cwd=checkout)
     git(
-        # -c arguments are instead of having to have git config --global nonsense
+        # -c arguments are instead of having to having to maintain stateful git config
         "-c",
-        "user.email=tech@opensafely.org",
+        "user.email=interactive@opensafely.org",
```

Comment:
Brian was concerned about the proliferation of email addresses when I asked earlier, so might be better to stick with tech@opensafely.org for now?

Reply:
I did that originally, but it produces a confusing info if you look at the commits in Github UI.

See the earlier commits here: https://github.com/bloodearnest/test-osi/commits/main

We already have a specific user that uses `tech@opensafely` as its email, so the Github UI matches to that github account, which is nothing to with OSI, and is confusing.

If the email doesn't match a github acccount, it instead shows the Name (see more recent commits), which is what we want in this case.

So, we need something other than `tech@opensafely.org`. What it is, and whether its a functional email address, doesn't actually matter, IMO. It's just do provide some form of identity for the committer, which is a bot essentially.

Reply:
If it doesn't need to be a functional email, then `interactive@opensafely.org` should be fine. I agree it does look better in the UI